src/backend/data/feed.js

```diff
@@ -165,15 +165,14 @@ class Feed {
 
   /**
    * Sets all stored feeds' lastModified + etag field to null. Used for production
-   * Returns a Promise
```

Comment:
You can leave this line, `async` functions implicitly return a Promise (good to call it out).